controller/space.go

```diff
@@ -382,6 +387,16 @@ func ConvertSpaceFromModel(request *goa.RequestData, p space.Space, additional .
 					Related: &relatedCodebasesList,
 				},
 			},
+			Workitems: &app.RelationGeneric{
```

Comment:
And could you align the naming of these. Stick to one pattern somehow. WorItemTypes, Workitems, Workitemlinktypes.